deps/libgit2.mk

```diff
@@ -28,6 +28,7 @@ endif
 endif
 ifeq ($(BUILD_OS),WINNT)
 LIBGIT2_OPTS += -G"MSYS Makefiles"
+LIBGIT2_OPTS += -DBUILD_CLAR=OFF -DUSE_SSH=OFF
```

Comment:
https://gist.github.com/Mikewl/9c644213ed1aa221c121b7ca201e629b

If `USE_SSH` is enabled then ldd returns the above result. Disabled, below. The above result is unusable in Julia even though in Cygwin it returns a perfectly valid result (but not in msys2).